README.md

```diff
@@ -143,7 +143,10 @@ or
 
 A basic ORT pipeline (using the _analyzer_, _scanner_ and _reporter_) can easily be run on
 [Jenkins CI](https://jenkins.io/) by using the [Jenkinsfile](./Jenkinsfile) in a (declarative)
-[pipeline](https://jenkins.io/doc/book/pipeline/) job.
+[pipeline](https://jenkins.io/doc/book/pipeline/) job. The job accepts various parameters that are translated to ORT
+command line arguments. An additional final parameter names an optional downstream job to trigger e.g. to further
```

Comment:
I believe the start of this sentence is a bit confusing: 'An additional final parameter names an optional downstream job to trigger'.

Reply:
I'm open to suggestions...

Reply:
Fair enough, I haven't made one as I wasn't really sure what you intended to say, but let's try. How about:

The job accepts various parameters that are translated to ORT
command line arguments. Additionally one can specify a downstream Jenkins job to be triggered before completion which in turn has the responsibility to copy the artifacts it needs from the upstream job.

Reply:
Ok, I've reworded the sentence based on your suggestion.  I've omitted the "before completion" part as it's not actually correct, see the `wait: false` build step configuration, i.e. the downstream job is called non-blocking.

Reply:
Oh know I know where my confusion comes from. Maybe it is just me, but it was due to the use of the word `names` next to the word `parameter` which lead me to the noun `parameter-names`. 

Consider swapping out `names` with `specifies` or something similar. Up to you.